assets/js/base/components/cart-checkout/product-variation-data/index.js

```diff
@@ -9,7 +9,7 @@ import classNames from 'classnames';
  * Returns a formatted element containing variation details.
  */
 const ProductVariationData = ( { className, variation = [] } ) => {
-	if ( ! variation ) {
+	if ( ! variation || variation.length === 0 ) {
```

Comment:
Why this change? Just curious.

Reply:
Ah, I should have explained that. It's a small change to avoid an empty `div` to appear.

Before this change, when `variation` was an empty array, this component was rendering `<div class="wc-block-components-product-variation-dat"></div>`. With the check added in this PR, we verify the array has some items before rendering the `div`.